app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabs.kt

```diff
@@ -141,26 +155,60 @@ fun SyncedTabsList(
 }
 
 /**
- * Text header for sections of synced tabs
+ * A non-collapsible text header for sections of synced tabs
+ *
+ * @param sectionText The section title for a group of synced tabs.
+ */
+@Composable
+fun SyncedTabsSectionHeader(sectionText: String) {
+    SyncedTabsCollapsibleSectionHeader(sectionText = sectionText)
+}
+
+/**
+ * Collapsible header for sections of synced tabs
  *
- * @param deviceName The name of the user's device connected that has synced tabs.
+ * @param sectionText The section title for a group of synced tabs.
+ * @param collapsed Indicates whether the section of content is expanded or collapsed. If null, the
+ * Icon will be hidden.
+ * @param onClick Optional lambda for handling section header clicks.
  */
 @Composable
-fun SyncedTabsDeviceItem(deviceName: String) {
+fun SyncedTabsCollapsibleSectionHeader(
```

Comment:
```suggestion
fun SyncedTabsSectionHeader(
```

Just dropping the collapse wording everywhere. The header doesn't technically collapse (compare to its content) was my other reasoning.